test/e2e/dra/deploy.go

```diff
@@ -419,6 +426,7 @@ func (d *Driver) SetUp(nodes *Nodes, resources Resources, devicesPerNode ...map[
 		} else {
 			fileOps.NumDevices = numDevices
 		}
+		listenerPort := int32(9000)
```

Comment:
We previously had two ports. Was this change intentional?

Reply:
This is now just the starting value for all ports.

Reply:
Let me add a comment...